lightningd/pay.c

```diff
@@ -1213,6 +1285,9 @@ static struct command_result *json_sendonion(struct command *cmd,
 		   p_opt("bolt11", param_string, &b11str),
 		   p_opt_def("msatoshi", param_msat, &msat, AMOUNT_MSAT(0)),
 		   p_opt("destination", param_node_id, &destination),
+#if EXPERIMENTAL_FEATURES
+		   p_opt("local_offer_id", param_sha256, &local_offer_id),
```

Comment:
I'd prefer not to introduce more underscore-delimited arguments, and shorter is better (`offerid` seems to fit our prior naming schema).

Reply:
offerid is a bit confusing though, since most payments are made in reference to an offer (and we'll all be driving flying cars, obv).

But this is only to link it to a *local* offer id.  I can elide underscores and make it localofferid?